### DISCUSSION TITLE
In LWT mode the circulation pump is activated every 3min 6 sec, wear and tear?

### DISCUSSION THREAD

**Discussion by buuhsmead** [Arnold-n/P1P2MQTT#121] [Q&A] [answered]
When I put the system in LWT mode, the circulation pump is activated every 3 minutes and 6 seconds.
That is, as I understand it, to check the LWT temperature.

Does this regular activation give a higher 'wear and tear'?
Is the system built to handle these 3 minutes and 6 seconds of activation for 10 years or more?

**Comment by Arnold-n** [comment, marked as answer]
To my knowledge this is normal if A2.1.9/[F-0D]=1, although in my system it is every 5 minutes. I do not think this is problematic, it's not the heat pump's compressor but only the circulation pump. However, it may be an indication that the system is "cycling" ("pendelen") which is not optimal for efficiency or compressor wear.
